config-templates/shopify/customer.yaml

```
@@ -0,0 +1,55 @@
+data_source:
+  name: shopify_customer
+  owners:
+    - support@transformdata.io
```

We don't actually want to be owners of these, right? Should we leave this out? It's optional in Metricflow anyway. 

Alternatively, you can leave it empty and then people will get an error and know to fill it in, something like this:

```suggestion
  owners:  # This will throw an error unless you add a list of owners or remove this entry.
```